index.html

```diff
@@ -0,0 +1,143 @@
+        <li><img class="startp-logo-nav mobile-hidden"
+            src="images/startup-business-conference-high-resolution-logo-color-on-transparent-background.png"
+            alt="company-logo"></li>
+        <li class="margin-top nav-item"><a href="about.html" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="#desktop-portfolio" class="nav-link">Program</a></li>
+        <li class="nav-item"><a href="#desktop-about" class="nav-link">Join</a></li>
+        <li class="nav-item"><a href="#desktop-contact" class="nav-link">Sponsor</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+        <li class="nav-item desktop-nav-item"><a href="#" class="nav-link">SBC Campaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </header>
+    <section>
+      <div class="hero-container">
+        <h1 class="Hero-H1 ">"Hello! Aspiring enterpreneues"</h1>
+        <span class="h1-paragraph">40 Startup Business Ideas
+          Conference 2022</span>
+      </div>
+      <div class="para-containerd">
+        <p class="hero-paragraph">
+          Join us in LAHORE for the "Startups 2022" confress to discover countless oppurtunities and benefits of
+          self-employment!
+        </p>
+      </div>
+      <h2 class="Date-container">
+        <span class="date">2022.11.25(FRI) ~ 27(SUN)</span>
+        <span class="location">@ Alhamra arts council, 68- Mall Road, Lahore</span>
+      </h2>
+    </section>
+  </article>
+  <!--------------------- The header section------------ -->
+
+  <!--------------------- The herosection------------ -->
+  <main>
+    <section class="Main-program">
+      <h2 class="main-program-heading">Main Program</h2>
+      <hr class="main-program-underline">
+      <article class="main-program-container">
+        <section class="main-program-section mp-section-1">
+          <img class="main-program-images" src="images/program_icon_01.png" alt="person standing at a podium">
+          <h3 class="secion-headings">Moivational Speeches</h3>
+          <p>Listen to the top enterpreneues from various countries
+          </p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_02.png" alt="Exhibition">
+          <h3 class="secion-headings">Exhibition</h3>
+          <p>Various new stratup opportunities for young enterpreneues</p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_03.png" alt="Forum">
+          <h3 class="secion-headings">Forum</h3>
+          <p>Have the time to share your thoughts with top enterpreneues</p>
+        </section>
+        <section class="main-program-section">
+          <img class="main-program-images" src="images/program_icon_04.png" alt="Workshop">
```

Comment:
- [ ] please make sure that the boxes should be transparent as the ones in the Design in order to match the design 
- Yours 
<img width="1223" alt="Screenshot 2022-11-24 at 18 11 17" src="https://user-images.githubusercontent.com/61790587/203838049-d0c2c20f-6423-4fd7-aa5a-18573ea9158a.png">
- Design 
<img width="857" alt="Screenshot 2022-11-24 at 18 11 11" src="https://user-images.githubusercontent.com/61790587/203838139-c4342d58-f79f-4d97-a63b-8c0d0e6d0c32.png">